code/script.js

```diff
@@ -1,9 +1,100 @@
 // A function that adds and remove the class "active" on the section you click on.
 // We haven't really talked about ´this´ yet, but we will... ;)
 // console.log(this) to see how it works
-function toggle() {
-  this.classList.toggle("active")
-}
+// function toggle() {
+//   this.classList.toggle("active")
+// }
 
 // Selects an HTML element, and calls a function which will be executed when the element is clicked.
-document.getElementById("section1").onclick = toggle
+// document.getElementById("section1").onclick = toggle
+
+
+// Sign in form validation
+const firstName = document.getElementById('firstName')
+const lastName = document.getElementById('lastName')
+const email = document.getElementById('email')
+const message = document.getElementById('message')
+const errorElement = document.getElementById('error')
+
+form.addEventListener('submit', (e) => {
+  let messages = []
+  if (firstName.value.length >= 15) {
+    messages.push('Name must be less than 15 characters')
+  }
+  if (email.value.length >= 20) {
+    messages.push('Email must be less than 20 characters')
+  }
```

Comment:
Try not to limit the number of characters of the email this way - mine won't work as an input even though it's a legit email address 😢